### ISSUE TITLE
Ability to edit entries made for networking under an Environment

### ISSUE THREAD

**Issue by awskaran** [data-dot-all/dataall#990]
**Is your idea related to a problem? Please describe.**
Under Admin > Environments > {SPECIFIC_ENV} > Environments tab once an entry is made for networking it cannot be edited.
The only way to correct a subnet id or vpc id is to delete the entry and make it again.

**Describe the solution you'd like**
I would like to have an ability to edit an existing networking entry under an environment.

*P.S. Don't attach files. Please, prefer add code snippets directly in the message body.*

**Comment by noah-paige** [comment]
Agreed - this would be helpful enhancement for networks in data.all. We would need to implement the following:

- New frontend view to update Network - `updateNetworkModal.js`
  - _CONSIDERATION_ : What aspects of network would it be helpful to edit? I am originally thinking `publicSubnetIds`, `privateSubnetIds`,`vpcId` and `tags` all should be editable?

- New API `updateNetwork` - query def in frontend and API specification in backend at `backend/dataall/core/vpc/api/` directory

- Resolver logic to interact with RDS DB and update values for the Network record


Let me know if you agree with the editable parameters @awskaran

I will work with the team to determine where we can prioritize the above in our backlog, thanks

**Comment by awskaran** [comment]
Yes @noah-paige I agree with the editable fields.
